app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::ItemsController < ApplicationController
+
+  def index
+    items = Item.paginate(page: what_page, per_page: per_page_count)
+    render json: serializer.new(items)
+  end
+  
+  def show
+    item = Item.find(params[:id])
+    render json: serializer.new(item)
+  end
+
+  def create
+    item = Item.create(item_params)
+    if item.save
+      render json: serializer.new(item), status: 201
+    end
+  end
+
+  def update
+    item = Item.find(params[:id])
+    item.update!(item_params)
+    render json: serializer.new(item)
+  rescue
+    render status: 404
+  end
+  
```

Comment:
I wanted to incorporate the `rescue` method instead of an `if/else` statement.  Could possibly be too generic and a case of `swallowing the exception`?